src/applications/personalization/preferences/tests/containers/PreferencesWidget.unit.spec.jsx

```diff
@@ -18,16 +18,23 @@ const props = {
 describe('<PreferencesWidget>', () => {
   it('should render empty view', () => {
     props.preferences.dashboard['education-training'] = false;
+    props.preferences.dashboard['health-care'] = false;
     const component = shallow(<PreferencesWidget {...props} />);
+    expect(component.find('Link').length).to.equal(1);
+    expect(component.find('Link').html()).to.contain('Select benefits now.');
```

Comment:
should this test check that the `Link` with `Find VA Benefits` is NOT rendered?

Reply:
The length check is intended to serve that purpose, but I've added that case as well.

Reply:
d'oh. thanks!